### PR TITLE
Punctuation, grammar, spelling, and formatting fixes

### PR DESCRIPTION
**Pull request by brandonlenz**
Trying not to forget about @kleinschmidtj's changes.

Opening up a PR from his branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,22 +204,22 @@ If you can stick to that goal, your existing unit tests will largely be transfer
 components that implement your form component will be straightforward:
 
 1. Create a copy of your custom Formik-aware component 
-   - Follow your project's naming and organizational conventions
-   - Know that the Formik-aware should be removed when the entire migration effort is completed (see [Cleanup](#cleanup) 
-   for more details)
+   - Follow your project's naming and organizational conventions.
+   - Know that the Formik-aware component should be removed when the entire migration effort is completed (see [Cleanup](#cleanup)
```
Comment (brandonlenz) [inline README.md:208]: This is a change I added

File: README.md [hunk 2]
```diff
@@ -265,14 +265,14 @@ until all unit tests pass
...
+| [`useFormikContext()`](https://formik.org/docs/api/useFormikContext) | [`useFormContext()`](https://react-hook-form.com/docs/useformcontext/) <br /> [`useFormState()`](https://react-hook-form.com/docs/useformstate/) <br /> [`useWatch()`](https://react-hook-form.com/docs/usewatch/)  | `useFormContext()` provides access to everything that your top-level `useForm()` hook provides. This is where most of your utilties are for imperatively interacting with the form. <br /> `useFormState()` provides meta information about the form's current state (like touched state, errors, submitCount, validation status, etc. <br /> `useWatch()` enables realtime access to a field's value (see [rendering](#rendering) for more details)  |
```
Comment (brandonlenz) [inline README.md:275]: Just white space changes here, to align all the table |'s

**Review by brandonlenz — COMMENTED** [review]
@kleinschmidtj I resolved conflicts with main, then made two small changes (Called out below). Otherwise, does this represent the edits you made? 

Thanks for doing this!

**Review by kleinschmidtj — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -204,22 +204,22 @@ If you can stick to that goal, your existing unit tests will largely be transfer
...
    - It is recommended to do this as soon as you are able to. 
    - Understandably, especially if context providers and mocking are involved, and you are new to React Hook Form, you
-   may not be able to do this confidently
+   may not be able to do this confidently.
    - If you use [Storybook](https://storybook.js.org/), it can also be a great way to manually sanity test your
-   component with a reasonable feedback loop
+   component with a reasonable feedback loop.
 4. Replace Formik hooks and utilities with the React Hook Form equivalents, extract the necessary return props, and 
-ensure they are properly wired into your resulting markup 
+ensure they are properly wired into your resulting markup .
```
[the remark this answers is not in the document]
Comment (brandonlenz) [reply README.md]: https://github.com/trussworks/formik-to-react-hook-form-migration-guide/pull/5/commits/cc8326ced4f951884f65cd82f8636f10b910d2d6

**Pull request merged**